rx888_source/CMakeLists.txt

```diff
@@ -9,14 +13,19 @@ else ()
     set(CMAKE_CXX_FLAGS "-O3 -std=c++17")
 endif ()
 
-include_directories("src/CyAPI")
-
 file(GLOB SRC "src/*.cpp")
-file(GLOB CYAPI "src/CyAPI/*.cpp")
 
 add_library(rx888_source SHARED ${SRC} ${CYAPI})
 target_link_libraries(rx888_source PRIVATE sdrpp_core)
 set_target_properties(rx888_source PROPERTIES PREFIX "")
 
+if (MSVC)
+  target_link_libraries(rx888_source PUBLIC Setupapi.lib)
```

Comment:
Why is this needed for a hardware module?

Reply:
Oh, I copy-paste this from somewhere